e2e/pages/adf/demo-shell/process-services/tasksCloudDemoPage.ts

```diff
@@ -157,15 +158,25 @@ export class TasksCloudDemoPage {
         await BrowserVisibility.waitUntilElementIsVisible(this.addActionTitle);
     }
 
-    async addAction(text: string): Promise<void> {
+    async addAction(text: string, disable: boolean): Promise<void> {
```

Comment:
this is a bad practice. A method should never have a boolean as input and switch from one behaviour to another based on the condition. Let's have a single responsibility and create 2 methods